src/core.js

```diff
@@ -524,8 +524,8 @@
           if (params.loginUrl) {
              connect.core.getPopupManager().clear(connect.MasterTopics.LOGIN_POPUP);
           }
-          connect.core.loginWindow = connect.core.getPopupManager().open(loginUrl, connect.MasterTopics.LOGIN_POPUP);
-
+          var options = params.loginPopup && params.loginPopup.forceWindow ? params.loginPopup : {}
```

Comment:
This will cause a null exception if loginPopup is `true`, you need to do a `instanceof` check here.

Reply:
If `params.loginPopup` is `true`, `params.loginPopup.forceWindow` is undefined, with no null exception. (You can test in FF/Chrome browser console with `true.testProp` or `true.testProp === undefined`)

The `params.loginPopup && params.loginPopup.forceWindow` will return false because `true && undefined` is `false`, so options will be `{}`.